### PR TITLE
Additional updates

### PR DESCRIPTION
**Pull request by bridgetkromhout**
More detail for the KEP.

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

File: keps/sig-network/563-dual-stack/README.md [hunk 1]
```diff
@@ -1347,22 +1347,27 @@ This capability will move to stable when the following criteria have been met.
...
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes. If we disable the feature gate, we must remove the CLI parameters. An
-  older client won't see or be able to use the new fields.
+  Yes. If you decide to turn off dual-stack after turning on, ensure all
+  services are converted to single-stack first (switch ipfamily to single-stack
+  on all services) and then disable the feature. Remove the CLI parameters, as
+  an older client won't see or be able to use the new fields. When the user
+  disables dual-stack from the controller manager, endpoints will no longer
```
Comment (aramase) [inline keps/sig-network/563-dual-stack/README.md]: New endpoints will no longer carry two sets. Existing endpoints may not be updated.

File: keps/sig-network/563-dual-stack/README.md [hunk 2]
```diff
@@ -1499,6 +1519,21 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
+
+  Once a service is saved (with the appropriate feature flag and one or more
+  cidrs), the controller manager scans the number of pods (seeing dual or
+  single) and creates endpoint objects pointing to these pods.Each node has
+  a proxy that watches for services, endpoints, endpoint slices, per family.
+  If the existing rules are removed but new ones don't resolve correctly, then
+  it will resolve on the next loop.
+
+  If allocation fails, that would translate to the pod not running if there is
+  no pod allocated from the cidr. If the pod is created but routing is not
```
Comment (aramase) [inline keps/sig-network/563-dual-stack/README.md]: no IP allocated from the CIDR

File: keps/sig-network/563-dual-stack/README.md [hunk 3]
```diff
@@ -1499,6 +1519,21 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
+  working correctly, there will be an error in the kube-proxy logs, so
+  debugging would take place by looking at iptables.  The cluster IP allocation
```
Comment (aramase) [inline keps/sig-network/563-dual-stack/README.md]: `looking at iptables similar to how it's already being done today`?

File: keps/sig-network/563-dual-stack/README.md [hunk 4]
```diff
@@ -1499,6 +1519,21 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
+  is allocated in a syncronous process; if this fails you'll see an error in
```
Comment (aramase) [inline keps/sig-network/563-dual-stack/README.md]: If this fails, the service creation will fail and the service object will not be persisted

**Review by lachie83 — APPROVED** [review]

File: keps/sig-network/563-dual-stack/README.md [hunk 5]
```diff
@@ -1499,6 +1519,22 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
        reporting dual-stack addresses.
 
+  *  Possible workload imbalances
+
+  Once a service is saved (with the appropriate feature flag and one or more
+  cidrs), the controller manager scans the number of pods (seeing dual or
+  single) and creates endpoint objects pointing to these pods.Each node has
+  a proxy that watches for services, endpoints, endpoint slices, per family.
+  If the existing rules are removed but new ones don't resolve correctly, then
+  it will resolve on the next loop.
```
Comment (khenidak) [inline keps/sig-network/563-dual-stack/README.md:1529]: worth mentioning that is the current behavior, not new

**Pull request merged**